### PR TITLE
updates stats

### PR DESCRIPTION
**Pull request by damencho**
- fix: Formatting around logging.
- feat: Moves some stats to Statistics.
- feat: Introduces new transcriber stats around errors.

### CODE REVIEW

File: src/main/java/org/jitsi/jigasi/stats/Statistics.java [hunk 1]
```diff
@@ -297,6 +357,10 @@ public static synchronized void sendJSON(
         stats.put(TOTAL_CALLS_JVB_NO_MEDIA, totalCallsJvbNoMedia.get());
         stats.put(TOTAL_CALLS_NO_HEARTBEAT, totalCallsWithNoHeartBeatResponse.get());
 
+        stats.put(TOTAL_TRANSCRIBER_STARTED, totalTrasnscriberStarted.get());
+        stats.put(TOTAL_TRANSCRIBER_STOPPED, totalTrasnscriberStopped.get());
+        stats.put(TOTAL_TRANSCRIBER_FAILED, totalTrasnscriberFailed.get());
+
```
Comment (bgrozev) [inline src/main/java/org/jitsi/jigasi/stats/Statistics.java:411]: totalTrasnscriberRequests and total15sIntervals don't get exposed here?

File: src/main/java/org/jitsi/jigasi/transcription/AbstractTranscriptPublisher.java [hunk 2]
```diff
@@ -845,46 +844,7 @@ private void maybeExecuteBashScripts()
...
-                                if (logger.isDebugEnabled())
-                                {
-                                    logger.debug("thrown stat: " +
-                                        DD_ASPECT_SUCCESS
-                                    );
-                                }
-                            }
-                            else
-                            {
-                                dClient.increment(DD_ASPECT_FAIL);
```
Comment (bgrozev) [inline src/main/java/org/jitsi/jigasi/transcription/AbstractTranscriptPublisher.java:879]: This looks like a useful metric to have?

Comment (bgrozev) [reply src/main/java/org/jitsi/jigasi/transcription/AbstractTranscriptPublisher.java:879]: Also note the behavior changes, we no longer call `p.waitFor()`

File: src/main/java/org/jitsi/jigasi/transcription/GoogleCloudTranscriptionService.java [hunk 3]
```diff
@@ -582,11 +563,8 @@ synchronized void sessionEnded()
             // round up to 15 second intervals
             int intervals15s = 1 + (int) (summedTime  / INTERVAL_LENGTH_MS);
 
-            if (client != null)
-            {
-                client.count(ASPECT_INTERVAL, intervals15s);
-                client.count(ASPECT_TOTAL_REQUESTS, requestsCount);
-            }
+            Statistics.incrementTotalTranscriber15sIntervals(intervals15s);
+            Statistics.incrementTotalTranscriberRequests();
```
Comment (bgrozev) [inline src/main/java/org/jitsi/jigasi/transcription/GoogleCloudTranscriptionService.java:574]: This one should probably be incremented directly in `incrementRequestsCounter`, or else be incremented *by* `requestsCount`

File: src/main/java/org/jitsi/jigasi/stats/Statistics.java [hunk 4]
```diff
@@ -209,6 +234,41 @@ public class Statistics
...
+     * Total number of transcriptions failes.
+     */
+    private static CounterMetric totalTrasnscriberFailed = JigasiMetricsContainer.INSTANCE.registerCounter(
+            TOTAL_TRANSCRIBER_FAILED,
+            "Total number of failed transcriptions.");
+
+    /**
+     * The total number of 15 second intervals submitted to the Google API for transcription.
+     */
+    private static LongGaugeMetric total15sIntervals = JigasiMetricsContainer.INSTANCE.registerLongGauge(
```
Comment (bgrozev) [inline src/main/java/org/jitsi/jigasi/stats/Statistics.java:309]: Why the weird unit of "15 seconds"? Can't we just use seconds or minutes?

**Review by bgrozev — CHANGES_REQUESTED** [review]

File: src/main/java/org/jitsi/jigasi/transcription/AbstractTranscriptPublisher.java (hunk 2 above)
Comment (damencho) [reply src/main/java/org/jitsi/jigasi/transcription/AbstractTranscriptPublisher.java:879]: This is something that is not used a lot, especially in our deployments. 
And the waitFor is just for the process finishing to get the output.

File: src/main/java/org/jitsi/jigasi/transcription/GoogleCloudTranscriptionService.java (hunk 3 above)
Comment (damencho) [reply src/main/java/org/jitsi/jigasi/transcription/GoogleCloudTranscriptionService.java]: Done.

File: src/main/java/org/jitsi/jigasi/stats/Statistics.java (hunk 1 above)
Comment (damencho) [reply src/main/java/org/jitsi/jigasi/stats/Statistics.java:414]: Done.

File: src/main/java/org/jitsi/jigasi/stats/Statistics.java (hunk 4 above)
Comment (damencho) [reply src/main/java/org/jitsi/jigasi/stats/Statistics.java]: Done.

**Pull request merged**